packages/frontend/src/components/search/SearchBand.tsx

```diff
@@ -0,0 +1,100 @@
+import Image from "next/image";
+import Link from "next/link";
+import React, { useState } from "react";
+import { type Song } from "@prisma/client";
+
+interface SearchBandProps {
+  showBand: boolean | undefined;
+  searchString: string;
+  songList: Song[];
+  filteredSongList: Song[];
+  channelProfile: string;
+}
+
+const SearchBand: React.FC<SearchBandProps> = ({
+  showBand,
+  searchString,
+  songList,
+  filteredSongList,
+  channelProfile,
+}) => {
+  const [activeList, setActiveList] = useState(-1);
+  return (
+    <>
+      {showBand && (
+        <>
+          <Link
+            href={`/band/${filteredSongList[0]!
+              .original_band!.toLowerCase()
+              .replace(/ /g, "-")}`}
+            className={`${
+              activeList === -2 ? "bg-[#f5f5f6] shadow-md" : ""
+            } flex gap-5 border-b p-5 pl-7 sm:rounded-lg sm:border-2`}
+            onMouseEnter={() => {
+              setActiveList(-2);
+            }}
+            onMouseLeave={() => {
+              setActiveList(-1);
+            }}
+          >
+            <div className="relative h-[70px] w-[70px] overflow-hidden rounded-full md:h-[100px] md:w-[100px]">
+              <Image
```

Comment:
Theres a particular error on NextJS's  `<Image/>`, this is due to it not receiving `channelProfile` on first load as the variable is not returned from the API yet.. a race condition happened here as the `<Image/>` renders before `channelProfile` was rendered,

To fix this,
Add a check for `channelProfile` before rendering
Add a placeholder before it becomes meaningful to avoid layout shift